plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/CloudConnectorImportWizard.java

```diff
@@ -48,224 +48,227 @@
-	 */
-	public boolean performFinish() {
-		if (storeWizardPage.equals(getContainer().getCurrentPage())) {
-			if (storeWizardPage.getConnectorStore().getSelection()) {
-				return performFinishStore();
-			} else if (storeWizardPage.getFileSystem().getSelection()) {
-				return performFinishFileSystem();
-			}
-		} else if (removeWizardPage.equals(getContainer().getCurrentPage())) {
-			return performFinishRemove();
-		}
-		return false;
-	}
+    /**
+     * Importing connector zip file to Developer Studio either from fileSystem or connector store.
+     */
+    public boolean performFinish() {
+        if (storeWizardPage.equals(getContainer().getCurrentPage())) {
+            if (storeWizardPage.getConnectorStore().getSelection()) {
+                return performFinishStore();
+            } else if (storeWizardPage.getFileSystem().getSelection()) {
+                return performFinishFileSystem();
+            }
+        } else if (removeWizardPage.equals(getContainer().getCurrentPage())) {
+            return performFinishRemove();
+        }
+        return false;
+    }
 
-	/**
-	 * This method will download the connector zip file and extract it to the relevant location when user has selected
-	 * import from connector store option.
-	 */
+    /**
+     * This method will download the connector zip file and extract it to the relevant location when user has selected
+     * import from connector store option.
+     */
     private boolean performFinishStore() {
-        final List<Connector> selectedConnectors = new ArrayList<>();
-        for (TableItem tableItem : storeWizardPage.getTable().getItems()) {
-            if (tableItem.getChecked()) {
-                selectedConnectors.add((Connector) tableItem.getData());
-            }
+        // final List<Connector> selectedConnectors = new ArrayList<>();
+        // for (TableItem tableItem : storeWizardPage.getTable().getItems()) {
+        // if (tableItem.getChecked()) {
+        // selectedConnectors.add((Connector) tableItem.getData());
+        // }
+        // }
+        // Job downloadJob = new Job("Downloading Connectors") {
+        // @Override
+        // protected IStatus run(IProgressMonitor monitor) {
+        // int noOfConnectors = selectedConnectors.size();
+        // int count = 1;
+        // monitor.beginTask("Downloading connector", noOfConnectors);
+        // for (Connector connector : selectedConnectors) {
+        // monitor.subTask(count + " of " + noOfConnectors + " : "
+        // + connector.getAttributes().getOverview_name() + " connector");
+        // String downloadLink = connector.getAttributes().getOverview_downloadlink();
+        // downloadConnectorAndUpdateProjects(downloadLink);
+        // monitor.worked(1);
+        // count++;
+        // }
+        // monitor.done();
+        // return Status.OK_STATUS;
+        // }
+        // };
+        // downloadJob.schedule();
+        // Refresh project
```

Comment:
Why do we have commented code here? Can't we remove them?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/832